code/src/components/Header.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import "header.css";
+
+export const Header = () => {
+    return (
+        <div>
```

Comment:
Is there a reason that you use `div` over here? It could be done this way :

```
return (
    <header>
        <h1 className="title">New music releases</h1>
    </header>
)
```